src/classes/json_data.py

```diff
@@ -153,6 +153,10 @@ def read_from_file(self, file_path, path_mode="ignore"):
                 contents = f.read()
             if not contents:
                 raise RuntimeError("Couldn't load {} file, no data.".format(self.data_type))
+            
+            if os.path.splitext(file_path) and os.path.splitext(file_path)[1] == '.osp':
```

Comment:
Do we need to qualify this replacement? In other words, the only "content" that gets passed to this function is our *.osp project file.

Reply:
Or should we protect it by version and if this regex finds any matches... Something like this
```
if self.damage_re.search(contents) and self.version_re.search(contents):
```

Reply:
This approach makes more sense probably, because it limits the impact of this change to only project files from 2.6.0 which have at least 1 missing ", right?  However, I would rename `version_re` to something more specific, and create another `version_re_260` for 2.6.0 detection, etc...

Reply:
Sounds good. When debugging. I did see that `.project` files get passed through here. But checking version sounds good.